lib/Hakyll/Core/Runtime.hs

```diff
@@ -43,9 +45,19 @@ import           Hakyll.Core.Util.File
 import           Hakyll.Core.Writable
 
 
+factsKey :: [String]
```

Comment:
Why not keep this value close to where it's used, in `build`?

Reply:
Because it is now used in two top-level definitions: `run` and `build`.

Reply:
Oh, right, it's used in the unchanged part of `run`! :sweat_smile: Sorry, and thanks.